recurly.services.yml

```diff
@@ -3,42 +3,42 @@ services:
     class: Drupal\recurly\Access\RecurlyAccessLocalAccount
     tags:
       - { name: access_check, applies_to: _access_check_recurly_local_account }
-    arguments: ['@current_route_match']
+    arguments: ['@current_route_match', '@recurly.config_manager']
```

Comment:
Ah ok, so services are treated differently in regards to how their dependencies are injected. Got it!

:+1: